src/conversion/downcast-converters.js

```diff
@@ -787,9 +783,12 @@ export function wrap( elementCreator ) {
 			return;
 		}
 
+		const viewWriter = conversionApi.writer;
+		const viewSelection = viewWriter.document.selection;
+
 		if ( data.item instanceof ModelSelection || data.item instanceof DocumentSelection ) {
 			// Selection attribute conversion.
-			viewWriter.wrap( conversionApi.viewSelection.getFirstRange(), newViewElement, conversionApi.viewSelection );
+			viewWriter.wrap( viewSelection.getFirstRange(), newViewElement, viewSelection );
```

Comment:
`view.Writer#wrap` does not take third parameter anymore.

This also means that maybe this method now can be simplified. I'd check for that.